src/recensio/plone/content/review_exhibition.py

```diff
@@ -1,11 +1,129 @@
+from collective.z3cform.datagridfield.datagridfield import DataGridFieldFactory
+from collective.z3cform.datagridfield.row import DictRow
+from plone.app.z3cform.widget import RelatedItemsFieldWidget
+from plone.autoform import directives
+from plone.autoform.directives import widget
 from plone.dexterity.content import Item
 from plone.supermodel import model
+from recensio.plone import _
+from z3c.relationfield.schema import RelationChoice
+from z3c.relationfield.schema import RelationList
+from zope import interface
+from zope import schema
 from zope.interface import implementer
 
 
+class IExhibitingInstitutionRowSchema(interface.Interface):
+    name = schema.TextLine(
+        title=_("Ausstellende Institution (z. B. Museum)"),
+        required=False,
+    )
+    gnd = schema.TextLine(
+        title=_("GND der Ausstellenden Institution"),
+        required=False,
+    )
+
+
+class IYearsRowSchema(interface.Interface):
+    years = schema.TextLine(
+        title=_("Years"),
+        required=False,
+    )
+
+
+class IExhibitingOrganisationRowSchema(interface.Interface):
+    name = schema.TextLine(
+        title=_("Ausstellende Organisation (z. B. Stiftung)"),
+        required=False,
+    )
+    gnd = schema.TextLine(
+        title=_("GND der Ausstellenden Organisation"),
+        required=False,
+    )
+
+
+class IDatesRowSchema(interface.Interface):
+    place = schema.TextLine(
+        title=_("Ort"),
+        required=False,
+    )
+    runtime = schema.TextLine(
+        title=_("Laufzeit"),
+        required=False,
+    )
+
+
 class IReviewExhibition(model.Schema):
     """Marker interface and Dexterity Python Schema for ReviewExhibition."""
 
+    exhibiting_institution = schema.List(
+        title=_("Ausstellende Institution"),
+        value_type=DictRow(
+            schema=IExhibitingInstitutionRowSchema,
+        ),
+        required=False,
+    )
+    widget(exhibiting_institution=DataGridFieldFactory)
+
+    dates = schema.List(
+        title=_("Ausstellung"),
+        value_type=DictRow(
+            schema=IDatesRowSchema,
+        ),
+        required=False,
+    )
+    widget(dates=DataGridFieldFactory)
+
+    years = schema.List(
+        title=_("Ausstellungsjahr"),
+        value_type=DictRow(
+            schema=IYearsRowSchema,
+        ),
+        required=False,
+    )
+    widget(years=DataGridFieldFactory)
+
+    exhibiting_organisation = schema.List(
+        title=_("Ausstellende Organisation"),
+        value_type=DictRow(
+            schema=IExhibitingOrganisationRowSchema,
+        ),
+        required=False,
+    )
+    widget(exhibiting_organisation=DataGridFieldFactory)
+
+    # FIXME: This was using the custom GNDReferenceBrowserWidget
+    # Use the standard RelatedItemsFieldWidget for now
+    curators = RelationList(
+        title="Kurator / Mitwirkende",
+        default=[],
+        value_type=RelationChoice(vocabulary="plone.app.vocabularies.Catalog"),
```

Comment:
Another way to define such relations is: https://github.com/syslabcom/recensio.plone/blob/main/src/recensio/plone/behaviors/authors.py#L17
Not sure yet what the difference is.

Reply:
FWIW, I notice a difference, the way the field is defined in authors requires you to search, whereas with the other option you can search or browse.